.gitignore

```diff
@@ -26,3 +26,4 @@ api/obj
 
 !packages/repositories.config
 /api/EmpowerOps.Volition.Api.0.1.0.nupkg
+*.orig
```

Comment:
please remove and delete these files. I think this is the third time I've brought this up.

Reply:
will do